plcc/src/io/CifParser.java

```diff
@@ -83,10 +85,11 @@
     private static Float oCoordXf, oCoordYf, oCoordZf;
     private static int lastLigandNumPDB = 0; // used to determine if atom belongs to new ligand residue
     private static String lastChainID = ""; // s.a.
+    private static int lastRnaNumPDB = 0;
     private static String[] tmpLineData;
     private static String tmpModelID;
-    private static String chainNum = null;
-    private static String chainType = null;
+    private static String chainNum = null;  // identity of the current chain e.g. A
```

Comment:
I am a bit puzzled. chainNum sounds like the number of the chain (1, 2, 3...), but what does 'identitiy' mean? Do you mean 'identifier' = ID (A, B, C...)?